core/src/test/java/com/novoda/merlin/receiver/ConnectivityReceiverShould.java

```diff
@@ -48,19 +55,20 @@ public void notNotifyTheMerlinServiceOnNullIntents() throws Exception {
 
     @Test
     public void notNotifyTheMerlinServiceOnNonConnectivityIntents() throws Exception {
-        Intent intent = new Intent();
-        intent.setAction("notNotifyTheMerlinServiceOnNonConnectivityIntents");
```

Comment:
yep sorry. It's the `toString()` which returns null, but also every method is empty (for android classes in this unit testing jar), including the ctor, so the action is not set if you do `new Intent()`.